scripts/falco-driver-loader

```diff
@@ -256,11 +256,11 @@ load_kernel_module() {
 		exit $?
 	fi
 
-	if [ -n "$ENABLE_DOWNLOAD" ]; then
+	if [  "$ENABLE_DOWNLOAD" = "yes" ]; then
```

Comment:
Why this?

Reply:
@Spartan-65 I believe this is not necessary. Unless I'm missing something, the previous check (`-n "$ENABLE_DOWNLOAD"`) should work file. If so, could you revert this line?

Thanks